python/cuml/test/dask/test_random_forest.py

```diff
@@ -71,7 +68,7 @@ def _prep_training_data(c, X_train, y_train, partitions_per_worker):
 
 
 @pytest.mark.parametrize('partitions_per_worker', [1, 3])
```

Comment:
I think we can further improve on time required by reducing the number of tests. This can be done by passing only `partitions_per_worker=3`.